commands/stats.go

```diff
@@ -0,0 +1,21 @@
+package stats
+
+
+// HandleStats handles a stat command
+func HandleStats(i *dgo.InteractionCreate, s *dgo.Session) {
+// Gathers stats
+	// Gathers number of bots
+		numberOfBots := "holderoftheplaces"
```

Comment:
If you are adding a placeholder for a number 
do it like this 

numberOfBots := 0